pokemongo_bot/__init__.py

```diff
@@ -34,10 +34,15 @@ def start(self):
 
     def take_step(self):
         location = self.navigator.take_step()
-        cells = self.find_close_cells(*location[0:2])
+        self.work_cells(1, self.position[0:2])
```

Comment:
When you have a function like this, with named arguments, please use keywords to false them:

`self.work_cells(work_on_forts=1, location=self.position[0:2]`

Note: doesn't make any sense sending the location here is inside this method you have:

```
if (location is None): 
    location = self.position[0:2] 
```

It's better to simplify me method and remove the `position` parameters entirely.